index.css

```diff
@@ -701,3 +701,406 @@ body {
   margin-right: 20px;
 }
 
+@media screen and (min-width: 769px) {
```

Comment:
Your navbar is not matching the Figma template of your choosing. Kindly fix it. 
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177908594-efc5f9c0-a77b-4ab7-9431-c5728993b07f.png) | ![image](https://user-images.githubusercontent.com/33268927/177908653-fb00db9d-64e5-4bd1-87d1-1f2d853872ae.png) |